src/app/login-page/login-page.component.ts

```diff
@@ -1,24 +1,35 @@
 import { Component, OnInit } from '@angular/core';
 import { AuthService } from '../core/services/auth.service';
+import { UserInfo } from '../core/models';
+import { switchMap } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-login-page',
   templateUrl: './login-page.component.html',
   styleUrls: ['./login-page.component.less']
 })
 export class LoginPageComponent implements OnInit {
+  public isAuth: boolean = false;
   public username: string;
   public password: string;
   public isAuthentificated: boolean;
 
-  constructor(private authService: AuthService) { }
+  constructor(
+    private authService: AuthService,
+    private router: Router
+  ) { }
 
   public ngOnInit(): void {
     this.isAuthentificated = this.authService.isAuthentificated();
   }
 
   public loginUser(): void {
     const isLogged: boolean = this.authService.userLogin(this.username, this.password);
-    this.isAuthentificated = isLogged;
+    if (isLogged) {
```

Comment:
this check should be performed by a guard

Reply:
fixed